core/controllers/admin.py

```diff
@@ -635,6 +636,168 @@ def get(self):
+                if not user_services.can_review_translation_suggestions(
+                        user_id, language_code=language_code):
+                    raise self.InvalidInputException(
+                        '%s does not have rights to review translation in '
+                        'language %s.' % (username, language_code))
+                user_services.remove_translation_review_rights_in_language(
+                    user_id, language_code)
+            elif review_category == constants.REVIEW_CATEGORY_VOICEOVER:
+                if not user_services.can_review_voiceover_applications(
+                        user_id, language_code=language_code):
+                    raise self.InvalidInputException(
+                        '%s does not have rights to review voiceover in '
+                        'language %s.' % (username, language_code))
+                user_services.remove_voiceover_review_rights_in_language(
+                    user_id, language_code)
+            elif review_category == constants.REVIEW_CATEGORY_QUESTION:
+                if not user_services.can_review_question_suggestions(user_id):
+                    raise self.InvalidInputException(
+                        '%s does not have rights to review question.' % (
+                            username))
+                user_services.remove_question_review_rights(user_id)
+            else:
+                raise self.InvalidInputException(
+                    'Invalid review_category: %s' % review_category)
+
+            email_manager.send_email_to_removed_community_reviewer(
+                user_id, review_category, language_code=language_code)
+        else:
+            raise self.InvalidInputException(
+                'Invalid removal_type: %s' % removal_type)
+
+        self.render_json({})
+
+
+class CommunityReviewersHandler(base.BaseHandler):
+    """Handler to show the existing reviewers."""
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        review_category = self.request.get('review_category')
+        language_code = self.request.get('language_code', None)
+        if language_code is not None and not (
+                utils.is_supported_audio_language_code(language_code)):
+            raise self.InvalidInputException(
+                'Invalid language_code: %s' % language_code)
+        if review_category not in [
+                constants.REVIEW_CATEGORY_TRANSLATION,
+                constants.REVIEW_CATEGORY_VOICEOVER,
+                constants.REVIEW_CATEGORY_QUESTION]:
+            raise self.InvalidInputException(
+                'Invalid review_category: %s' % review_category)
+        usernames = user_services.get_community_reviewer_usernames(
+            review_category, language_code=language_code)
+        self.render_json({'usernames': usernames})
+
+
+class CommunityReviewerRightsDataHandler(base.BaseHandler):
```

Comment:
Ditto?

Reply:
This is going to be a right's dict of a specific user, so I've left it as-is.